build/build.py

```diff
@@ -813,7 +815,9 @@ def configureCompilerOptions(self):
             config['cxx']['optz_fastest']   = '-O3'
 
             gxxCompileFlags='-fPIC'
-            if cxxCompiler == 'g++' and gccHasCpp11():
+            botherChecking = self.env['cpp11support'] or self.env['force_cpp11support']
```

Comment:
I don't think you need `self.env['force_cpp11support']`.  If an individual project wants to force C++11 support, they can do so in their top-level wscript by putting `self.env['cpp11support']` in their configure section.  I don't think you need two separate environment variables.